ash/Cargo.toml

```diff
@@ -15,11 +15,13 @@ rust-version = "1.59.0"
 libloading = { version = "0.7", optional = true }
 
 [features]
-default = ["loaded", "debug"]
+default = ["loaded", "debug", "std"]
+# Whether ash depends on the standard library. Disabiling this raises the MSRV to 1.64
```

Comment:
`Disabiling`?

```suggestion
# Whether ash depends on the standard library. Disabling this raises the MSRV to 1.64.
```

And add a period at the end to match the other feature-flag comments.